package.json

```diff
@@ -789,6 +795,10 @@
                 {
                     "command": "postgreSQL.connectDatabase",
                     "when": "view == azureDatabasesExplorer && viewItem == postgresDatabase"
+                },
+                {
+                    "command": "postgreSQL.executeQuery",
```

Comment:
With this entry point, you assume the active editor is the query they want to execute, but it might not be the right query or even a query at all. I would prefer to remove this entry point - users should only be able to execute a query if they have a ".sql" file open. When in doubt, I would model this after mongo scrapbook behavior